addons/web/static/src/legacy/scss/report.scss

```diff
@@ -13,6 +13,11 @@ body {
     font-family: $o-default-report-font;
 }
 
+hr {
+    border-top: 1px solid !important;
+    border-top-color: rgb(0, 0, 0);
```

Comment:
As said in my comment, I think it's better to keep the original color that would be (if on white):

`rgb(229, 229, 229)`

or 

`rgb(230, 230, 230)`